samples/manifests/demo/demo-lake-creator-cfn-template.yaml

```diff
@@ -29,8 +29,6 @@ Resources:
         - ''
         - - 'orbit-'
           - !Ref envname
-          - '-'
-          - !Ref "AWS::Region"
```

Comment:
this stays, and the one below stays.  but this is the wrong file. make the following change in manifests/plugins/demo-lake-creator-cfn-template.yaml

on line 170, we should add 
- '-'
- !Ref "AWS::Region"

you'll s

Reply:
What is commited is backwards....we need this added:
          - '-'
          - !Ref "AWS::Region"